articles/event-hubs/resource-manager-exceptions.md

```diff
@@ -35,6 +35,7 @@ The following sections provide various exceptions/errors that are surfaced throu
 | ---------- | ------------- | ------------- | ----------- | -------------- |
 | BadRequest | 40000 | PartitionCount can't be changed for an event hub. | Basic, standard, or premium tier of Azure Event Hubs doesn't support changing partitions. | Create a new event hub with the wanted number of partitions in your basic, standard, or premium tier namespace. Partition scale-out is supported for [dedicated clusters](event-hubs-dedicated-overview.md). |
 | BadRequest | 40000 | The value '#' for MessageRetentionInDays isn't valid for the Basic tier. the value can't exceed '1' day(s). | Basic tier Event Hubs namespaces only support message retention of up to 1 day. | If more than one day of message retention is wanted, [create a standard Event Hubs namespace](event-hubs-create.md). | 
+| BadRequest | 40000 | Event Hub Entity cannot be disabled. | Capture feature is enabled for continous flowing of messages. | Disable Capture feature and then try disabling the event Hub entity and it will work. 
```

Comment:
@SudeshnaB1902 - please update azure-docs to azure-docs-pr (private repo) in the URL before you make updates and submit PRs. 

Please update it as shown below. Also, the description doesn't seem to answer the question. Do you mean "An event hub can't be disabled if the Capture feature is enabled for it" or something like that. 

| BadRequest | 40000 | The event hub can't be disabled. | The Capture feature is enabled for continuous flow of messages. | Disable the Capture feature and then try disabling the event hub. |

Reply:
Yes @spelluru An event hub cannot be disabled if the Capture feature is enabled . Sure will do it and raise the pr again